Mathlib/FieldTheory/Normal.lean

```diff
@@ -127,8 +127,8 @@ theorem AlgHom.normal_bijective [h : Normal F E] (ϕ : E →ₐ[F] K) : Function
     exact ⟨y, hy⟩⟩
 #align alg_hom.normal_bijective AlgHom.normal_bijective
 
--- Porting note: `[Field F] [Field E] [Algebra F E]` added by hand.
```

Comment:
This porting note looks unrelated. Did you mean to delete it? I see similar ones not deleted later in the file